fhir-search/src/main/java/com/ibm/fhir/search/uri/UriBuilder.java

```diff
@@ -107,6 +99,21 @@ public String toSearchSelfUri() throws URISyntaxException {
         return selfUri.toString();
     }
 
+    /**
+     * Appends the search parameters to the query string, if joining them together results in a non-empty string.
+     */
+    private void appendSearchParameters() {
+        if (!context.getSearchParameters().isEmpty()) {
+            Predicate<String> stringIsEmpty = String::isEmpty;
+            String searchParameters = context.getSearchParameters().stream().map(p -> serializeSearchParmToQueryString(p))
+                    .filter(stringIsEmpty.negate()).collect(Collectors.joining(SearchConstants.AND_CHAR_STR));
```

Comment:
minor style suggestion
```suggestion
            String searchParameters = context.getSearchParameters().stream()
                    .map(p -> serializeSearchParmToQueryString(p))
                    .filter(stringIsEmpty.negate())
                    .collect(Collectors.joining(SearchConstants.AND_CHAR_STR));
```

Reply:
Updated.